config/multitenancy.php

```diff
@@ -11,6 +11,11 @@
 use Spatie\Multitenancy\Models\Tenant;
 
 return [
+    /*
+     * If your application uses Laravel Octane, set it to `true`.
+     */
+    'octane' => false,
```

Comment:
It would be cool we could just detect Octane. @themsaid is there something in Laravel/Octane that allows easy detection?

Reply:
You can do `isset($_SERVER['LARAVEL_OCTANE'])` to check if it's running under Octane.

Reply:
Thank you